pages.es/common/chown.md

```diff
@@ -0,0 +1,23 @@
+# chown
+
+> Cambia la propiedad de usuario y grupo sobre archivos y directorios.
+
+- Cambia el usuario propietario de un archivo/directorio:
+
+`chown {{usuario}} {{ruta/hacia/archivo_o_directorio}}`
+
+- Cambia el usuario y grupo propietario de un archivo/directorio:
+
+`chown {{usuario}}:{{grupo}} {{ruta/hacia/archivo_o_directorio}}`
+
+- Cambia de forma recursiva la propiedad sobre un directorio y su contenido:
```

Comment:
```suggestion
- Cambia de forma recursiva el propietario sobre un directorio y su contenido:
```

Reply:
In this case the change is less necessary, but it may be more understandable as you say.